sample/src/main/AndroidManifest.xml

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.bucketeer.sample"
```

Comment:
Using `package="io.bucketeer.sample" in `AndroidManifest.xml` is no longer recommended after AGP 8.0.0